Lib/test/test_future.py

```diff
@@ -275,6 +275,9 @@ def test_annotations(self):
         eq("dict[str, int]")
         eq("set[str,]")
         eq("tuple[str, ...]")
+        eq("tuple[(str, *types)]")
+        eq("tuple[str, int, (str, int)]")
```

Comment:
This case was passing before. Did you meant `tuple[str, int, *(str, int)]`

Reply:
It is for validating tuples have parens when they are inside of another tuple with different precedence.